src/extensions/model-archive/quality-assessment/prop.ts

```diff
@@ -18,17 +18,35 @@ import { AtomicIndex } from '../../../mol-model/structure/model/properties/atomi
 import { CustomPropSymbol } from '../../../mol-script/language/symbol';
 import { Type } from '../../../mol-script/language/type';
 import { QuerySymbolRuntime } from '../../../mol-script/runtime/query/compiler';
-import { ParamDefinition as PD } from '../../../mol-util/param-definition';
+import { ParamDefinition, ParamDefinition as PD } from '../../../mol-util/param-definition';
 
 export { QualityAssessment };
 
 interface QualityAssessment {
-    localMetrics: Map<string, Map<ResidueIndex, number>>
+    local: QualityAssessment.Local[]
+    /** id -> metric info */
+    localMap: Map<number, QualityAssessment.Local>
+
+    // default pLDDT and qmean metrics
     pLDDT?: Map<ResidueIndex, number>
     qmean?: Map<ResidueIndex, number>
+
+    // NOTE: Keeping this around in case someone is using it
+    // TODO: Remove in Mol* 5.0
+    localMetrics: Map<string, Map<ResidueIndex, number>>
```

Comment:
`
```suggestion
    /**
     * @deprecated
     * NOTE: Keeping this around in case someone is using it
     * TODO: Remove in Mol* 5.0
     */
     localMetrics: Map<string, Map<ResidueIndex, number>>
```